nox/_options.py

```diff
@@ -225,19 +233,43 @@ def _posargs_finalizer(
     return posargs[dash_index + 1 :]
 
 
+def _python_completer(
+    prefix: str, parsed_args: argparse.Namespace, **kwargs: Any
+) -> Iterable[str]:
+    global_config = parsed_args
```

Comment:
Is it truly necessary to have this argument?

Reply:
I don't think so, not sure why I opted to do it. Removed in 49634fcb169165f1a822a97272ee6628d258e553.